datalad/distribution/tests/test_update.py

```diff
@@ -756,8 +756,7 @@ def check_merge_follow_parentds_subdataset_detached(on_adjusted, path):
     # the explicit revision fetch fails.
     (ds_src_s1.pathobj / "bar").write_text("bar content")
     ds_src.save(recursive=True)
-    ds_src_s1.repo.checkout(
-        ds_src_s1.repo.get_corresponding_branch("master"))
+    ds_src_s1.repo.checkout('master')
```

Comment:
Eh, that was silly of me.  I guess in some earlier iteration that was a plain `.get_corresponding_branch()`, and I didn't drop the call when I switched to explicitly using "master", or something :x